templates/share/site_header.html

```diff
@@ -23,18 +23,20 @@
                     </a>
                 </div>
                 <div class="collapse navbar-collapse" id="main-navbar-collapse">
-                <ul class="nav navbar-nav navbar-left">
+                <ul class="navbar-nav navbar-left">
```

Comment:
Removing this nav tag might be a problem, because this is a part of the code which makes the menu work on mobile. Can you get this to work without removing it?

Reply:
After the change to make the dialog work, I was able to make it work with the nav.